app/controllers/settings.js

```diff
@@ -2,16 +2,19 @@ import Ember from 'ember';
 
 export default Ember.Controller.extend({
   envVars: Ember.computed.filterBy('model.envVars', 'isNew', false),
-  cronJobs: Ember.computed.sort('model.cronJobs', function(a, b) {
-    if(a.get('branch.defaultBranch')) {
-      return -1;
-    } else if(b.get('branch.defaultBranch')) {
-      return 1;
-    } else {
-      return a.get('branch.name') > b.get('branch.name');
-    }
-  }),
-  branches: Ember.computed.sort('model.branches', function(a, b) {
+  freeBranches: function() {
+    var cronJobs = this.get('model.cronJobs.jobs');
+    var branches = this.get('model.branches').filter(function(branch) {
+      return branch.get('exists_on_github');
+    });
+    return branches.filter(function(branch) {
+      return ! cronJobs.any(function(cron) {
+        return branch.get('name') === cron.get('branch').get('name');
+      });
+    });
+  }.property('model.cronJobs.jobs.@each'),
```

Comment:
Ember land is moving away from this computed property syntax and more toward the `Ember.computed`-type used elsewhere. It seems to be widely agreed that this syntax is confusing, and it requires “prototype extensions” that are falling out of favour. So my advice is to change this to just `Ember.computed('model.cronJobs.jobs.@each', function() ` up on the line where the property is declared. But this syntax appears elsewhere in the codebase so it’s not something I’m insisting upon, just a direction I’d like the codebase to move.